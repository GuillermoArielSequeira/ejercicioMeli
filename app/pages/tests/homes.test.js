const React = require('react');
const { shallow } = require('enzyme');
const Home = require('../home');

describe('Home Component', () => {
  it('render Home correctly', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  })
});
