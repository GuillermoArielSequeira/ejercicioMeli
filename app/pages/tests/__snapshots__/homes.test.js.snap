// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component render Home correctly 1`] = `ShallowWrapper {}`;
